File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-03/plagiarized/L2/09/Level2.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function calculateBMI() {
  rl.question('Enter weight in pounds: ', (weightStr) => {
    const weight = parseFloat(weightStr);
    rl.question('Enter feet: ', (feetStr) => {
      const feet = parseFloat(feetStr);
      rl.question('Enter inches: ', (inchesStr) => {
        const inches = parseFloat(inchesStr);
        const height = feet * 12 + inches;
        const bmi = weight * 0.45359237 / ((height * 0.0254) * (height * 0.0254));
        console.log(`BMI is ${bmi}`);
        if (bmi < 18.5) {
          console.log('Underweight');
        } else if (bmi < 25) {
          console.log('Normal');
        } else if (bmi < 30) {
          console.log('Overweight');
        } else {
          console.log('Obese');
        }
        rl.close();
      });
    });
  });
}

calculateBMI();

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-03/plagiarized/L2/09/Level2.kt
package IR

import java.util.*

/*
* To change this license header, choose License Headers in Project Properties.
* To change this template file, choose Tools | Templates
* and open the template in the editor.
*/
/**
 *
 * @author FD8DBE9073AFCC0504CD6901E1091CAD
 */
object Level2 {
    @JvmStatic
    fun main(args: Array<String>) {
        val `in` = Scanner(System.`in`)

        print("Enter weight in pounds: ")
        val berat = `in`.nextDouble()
        print("Enter feet: ")
        val feet = `in`.nextDouble()
        print("Enter inches: ")
        val inches = `in`.nextDouble()
        val tinggi = feet * 12 + inches

        val bmi = berat * 0.45359237 / ((tinggi * 0.0254) * (tinggi * 0.0254))

        println("BMI is $bmi")
        if (bmi < 18.5) println("Underweight")
        else if (bmi < 25) println("Normal")
        else if (bmi < 30) println("Overweight")
        else println("Obese")
    }
}
